packages/cozy-clisk/src/launcher/saveFiles.js

```diff
@@ -221,6 +231,81 @@ async function ensureAllDestinationFolders({
   for (const path of pathsList) {
     await fileCollection.ensureDirectoryExists(path)
   }
+
+  const slug = options?.manifest?.slug
+  const { included: existingKonnectorFolders } = await client.query(
+    Q('io.cozy.files')
+      .partialIndex({ type: 'directory', trashed: false })
+      .referencedBy({
+        _type: 'io.cozy.konnectors',
+        _id: `io.cozy.konnectors/${slug}`
+      })
+  )
```

Comment:
Be careful cf https://github.com/cozy/cozy-client/issues/1216 (cc @paultranvan )